wolox-books/src/app/services/user.service.ts

```diff
@@ -0,0 +1,17 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { User } from '../models/user';
+
+@Injectable({
+  providedIn: 'root'
+})
+
+export class UserService {
+  constructor(private httpClient: HttpClient) {}
+
+  readonly SIGN_UP_URL = 'https://books-training-rails.herokuapp.com/api/v1/users';
```

Comment:
the base url ('https://books-training-rails.herokuapp.com/api/v1') should be a environment variable. If we have a different environment this url will change